jekyll/_cci2/index.md

```diff
@@ -1,24 +1,49 @@
 ---
 layout: classic-docs
-title: "Welcome to CircleCI 2.0"
+title: "2.0 Docs (Beta)"
 description: "Landing page for CircleCI 2.0"
 permalink: /2.0/
 ---
 
-Thanks for your interest in CircleCI 2.0!
+Welcome to CircleCI 2.0 Beta! The Beta release of CircleCI 2.0 includes many improvements for performance, default Docker setup, and customization. 
 
-We are currently in Beta, so lots will be changing here. We LOVE it when you make suggestions (or, even better, make pull requests on the <a href="{{ site.gh_url }}">open source repo</a> that powers these docs) that address any errors or shortcomings you find here.
+## Programming Language Support
 
-## If you're completely new to CircleCI...
+Code that builds on Linux will generally build on CircleCI 2.0. For some language versions, CircleCI provides demo applications and Docker images for primary container use, as follows: 
 
-Read about [what we do]({{ site.baseurl }}/2.0/about-circleci/) and follow up with your [first steps]({{ site.baseurl }}/2.0/first-steps/).
+- Clojure 1.8 and later, see https://github.com/circleci/cci-demo-clojure and https://hub.docker.com/r/circleci/clojure
+- Go 1.7 and later, see https://github.com/circleci/cci-demo-go and https://hub.docker.com/r/circleci/golang
+- Node.js 4 and later, see https://github.com/circleci/cci-demo-react and https://hub.docker.com/r/circleci/node
+- PHP 5 and later, see https://github.com/circleci/cci-demo-lumen and https://hub.docker.com/r/circleci/php
+- Python 2 and later, see https://github.com/circleci/cci-demo-flask and https://hub.docker.com/r/circleci/python
+- Ruby 2 and later, see https://github.com/circleci/cci-demo-rails and https://hub.docker.com/r/circleci/ruby 
 
-## If you're familiar with CircleCI...
+In addition, CircleCI provides Docker database images for use as a secondary service container:
 
-Check out the major changes in 2.0 and learn how to [migrate from 1.0]({{ site.baseurl }}/2.0/migrating-from-1-2/).
+- MongoDB 3 and later, see https://hub.docker.com/r/circleci/mongo
+- MySQL 5 and later, see https://hub.docker.com/r/circleci/mysql
+- PostgreSQL 9 and later, see https://hub.docker.com/r/circleci/postgres
 
-Do you just want to see working configuration for a [sample project]({{ site.baseurl }}/2.0/project-walkthrough/)? Or maybe you have a [specific language or framework]({{ site.baseurl }}/2.0/demo-apps/) in mind? Wherever you start, we’re thrilled to have you here.
+Build projects in C, C#, C++, Clojure, Elixir, Erlang, Go, Groovy, Haskell, Haxe, Java, Javascript, Node.js, Perl, PHP, Python, Ruby, Rust, Scala and many more. 
 
-Happy building!
+## 2.0 Performance Improvements
+
+- Infrastructure upgrades resulting in significant performance improvements that reduce overall time to build, test, and deploy for all platforms at scale. Refer to the [CircleCI Pricing](https://circleci.com/pricing/) page for build container pricing or to contact a sales representative. 
```

Comment:
Oh right, thanks, I will get rid of this and replace it with the beta signup page link.